README.md

```diff
@@ -75,6 +82,41 @@ kernel-build-container   gcc-5      ed6270a767c6   8 hours ago          720MB
 kernel-build-container   gcc-4.9    408db89527ec   8 hours ago          913MB
 ```
 
+### Building all containers (manage_containers.py compact alternative):
+
+```console
+python3 manage_containers.py -a all -q
```

Comment:
Also, please add other usage examples of this tool to the README.
Thanks!

Reply:
Expanded a bit :+1: 

https://github.com/a13xp0p0v/kernel-build-containers/pull/22/commits/3cae5e9e421255fdce98316e0ed463609fb271b5